### PR TITLE
[MCKIN-10516] Bump `xblock-drag-and-drop-v2` to a temporary version

### PR DESCRIPTION
**Pull request by Agrendalath**
This is a version bump PR for edx-solutions/xblock-drag-and-drop-v2#210.

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @Agrendalath! I've created [OSPR-3458](https://openedx.atlassian.net/browse/OSPR-3458) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

File: requirements/edx/custom.txt [hunk 1]
```diff
@@ -4,8 +4,8 @@
 -e git+https://github.com/edx-solutions/xblock-mentoring.git@8837eb5d91fed05ec4758dfd9b9e7adc5c906210#egg=xblock-mentoring
 -e git+https://github.com/edx-solutions/xblock-image-explorer.git@v1.1.4#egg=xblock-image-explorer==1.1.4
 -e git+https://github.com/edx-solutions/xblock-drag-and-drop.git@92ee2055a16899090a073e1df81e35d5293ad767#egg=xblock-drag-and-drop
-# FIXME: temp hash, until a proper solution is findout (ref ticket: https://edx-wiki.atlassian.net/browse/MCKIN-8249)
--e git+https://github.com/edx-solutions/xblock-drag-and-drop-v2.git@28c82e3d63f011029343115cb1b07e27afaeeb3e#egg=xblock-drag-and-drop-v2==2.2.2
+# FIXME: temp hash, until (https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/207 is merged into `master`)
```
Comment (xitij2000) [inline requirements/edx/custom.txt]: The situation is a lot more complex now, since there is greater divergence between the branches.

Comment (Agrendalath) [reply requirements/edx/custom.txt]: Should we create a new branch from `28c82e3d63f011029343115cb1b07e27afaeeb3e` and cherry-pick this change there?

Comment (xitij2000) [reply requirements/edx/custom.txt]: There is a task to reconcile the branches, it just needs to be prioritised.

**Pull request merged**